### ISSUE TITLE
Double-click on map opens info window

### ISSUE THREAD

**Issue by eightysteele** [MapofLife/MOL#65]
If you double-click to zoom on the map, if your clicks hit a layer, it opens the info window. We don't want to open the info window on a double-click.

**Comment by jmalczyk** [comment]
This is really an issue in CartoDB's code. I submitted a ticket there, moving to a lower priority while we wait on them.

**Comment by jmalczyk** [comment]
[cartodb-mapsv3 #12](https://github.com/Vizzuality/cartodb-gmapsv3/issues/12)

**Comment by eightysteele** [comment]
Cool. You can also fork their repo, fix this, then send a pull request.
